Library/Homebrew/cmd/update.sh

```diff
@@ -375,11 +375,21 @@ user account:
 EOS
   fi
 
+  # we may want to use Homebrew CA certificates
+  if [[ -n "${HOMEBREW_FORCE_BREWED_CA_CERTIFICATES}" && ! -f "${HOMEBREW_PREFIX}/etc/ca-certificates/cert.pem" ]]
+  then
+    # we cannot install Homebrew CA certificates if homebrew/core is unavailable.
+    if [[ -d "${HOMEBREW_LIBRARY}/Taps/homebrew/homebrew-core" || -n "${HOMEBREW_INSTALL_FROM_API}" ]]
+    then
+      brew install ca-certificates || true
```

Comment:
the `|| true` seems undesirable here, we want to handle if this fails (with at least a message), no?

Reply:
I had a reason in my head before but so much has changed that I can't remember it and it probably no longer applies.

I'm not sure what the message would be though. The solution for brewed curl and git is to install it yourself and put it in the PATH. This doesn't apply to CA certificates.

Reply:
I think it's worth just failing in this case, then.